pyproject.toml

```diff
@@ -62,6 +62,7 @@ pymdown-extensions = "^6.1"
 lxml = "^4.4"
 ipython = "^7.10"
 markdown-include = "^0.5.1"
+aiohttp-socks = "^0.3.4"
```

Comment:
*aiohttp-socks* is also should be added to `[tool.poetry.dependencies]` block as optional dependency (See `uvloop`) and in block `[tool.poetry.extras]` as `proxy = ["aiohttp-socks"]`